### PR TITLE
feat: Update the frontpage and user docs

### PR DESCRIPTION
**Pull request by Jelledb**

### CODE REVIEW

File: docs/meshcloud.index.md [hunk 1]
```diff
@@ -2,56 +2,68 @@
...
-| [meshTenant](./meshcloud.tenant.md)            | An isolated environment in a specific cloud platform, e.g. an AWS Account.                |
-| [meshLandingZone](meshcloud.landing-zones.md)  | Defines configuration and governance policies for cloud environments.                     |
-| [meshPlatform](meshcloud.platforms.md) | An individual cloud platform connected to meshcloud.                                      |
-| [meshLocation](meshcloud.platforms.md) | A grouping of meshPlatforms, e.g. by geographic region.                                   |
-
+Welcome to the meshStack documentation! This part of the documentation is intended for end users
+of the meshStack platform. Be aware that meshStack might be rebranded in your organization and has a different name,
+but everything you find in the documentation will apply to the meshStack you are using as well.
+
+meshStack is useful for anyone that wants to use cloud services in their organization and manage their costs.
```
Comment (chewbeccy) [inline docs/meshcloud.index.md:12]:
```suggestion
meshStack is useful for anyone that wants to use platform services in their organization and manage their costs.
```

File: docs/meshcloud.index.md [hunk 2]
```diff
@@ -2,56 +2,68 @@
...
+Maybe you are a DevOps team that wants to create a few cloud projects in Microsoft Azure their projects across
```
Comment (chewbeccy) [inline docs/meshcloud.index.md:13]:
```suggestion
Maybe you are a Application team that wants to create a few cloud projects in Microsoft Azure their projects across
```

File: docs/meshcloud.index.md [hunk 3]
```diff
@@ -2,56 +2,68 @@
...
+  you invite team members that are needed for that environment, e.g. for a production project you will invite your SRES. Within
+  the project, you will have one or more cloud tenants.
+- [Tenant](./meshcloud.tenant.md): The tenant is an isolated and tangible place in an actual cloud provider. It depends on what cloud
+  you use what this maps to. For example for AWS this maps to an `Account`, for Azure this is a `Subscription` and for Google Cloud
+  it is a `Project`. meshStack can easily let you log in and redirect you to the tenant in the respective cloud provider.
+- [Landing Zone](./meshcloud.landing-zones.md): The landing zone is a template created by your organization that you pick
+  when you create a new tenant. It cannot be changed afterward. The idea of the landing zone is that you already get a set
+  of policies and perhaps resources to start using the cloud right away according to the compliance and best practices
+  of your organization.
+- [Building Blocks](./administration.building-blocks.md): A building block is a piece of infrastructure that can be booked
```
Comment (chewbeccy) [inline docs/meshcloud.index.md:37]: I wonder if this is always infrastructure

File: docs/meshcloud.index.md [hunk 4]
```diff
@@ -2,56 +2,68 @@
...
 
-   Get an overview of all locations and their platforms of your [meshProject](./meshcloud.project.md). Add new team members to it or change the payment methods of your project.
+- [project control plane](./meshcloud.project-resources.md#project-control-plane): Get an overview of all tenants of your [meshProject](./meshcloud.project.md). Add new team members to it or change the payment methods of your project.
 
-- [tenant control plane](./meshcloud.project-resources.md#tenant-control-plane)
+- [tenant control plane](./meshcloud.project-resources.md#tenant-control-plane): Access the platform of your [meshTenant](./meshcloud.tenant.md), request Quotas for it or check the [usage reports](./meshcloud.project-metering.md#tenant-usage-reports).
 
-    Access the platform of your [meshTenant](./meshcloud.tenant.md), request Quotas for it or check the [usage reports](./meshcloud.project-metering.md#tenant-usage-reports).
+The above description serve as an example what you are able to do as Workspace Manager. Workspace Members will have a lot less
```
Comment (chewbeccy) [inline docs/meshcloud.index.md:54]:
```suggestion
The above description serves as an example what you are able to do as Workspace Manager. Workspace Members will have a lot less
```

**Review by chewbeccy — APPROVED** [review]
just some smaller things. I think in a second step we need to have a look at platforms vs cloud platforms. But this is something that is part of the running CR and discussions regarding out product direction

File: docs/meshcloud.index.md (hunk 3 above)
Comment (Jelledb) [reply docs/meshcloud.index.md:37]: It is based on Terraform and Terraform is an infrastructure-as-code tool so I think it's the best explanation we can have,

**Pull request merged**